runtime-builder/tests/integration/go-get.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd $(dirname $0)
+export GOPATH=$(pwd -P)
+go get -d -tags appenginevm app 
+find . -name .git -type d | xargs rm -rf {} \;
```

Comment:
Sorry, I just remember now that I use {} with -exec instead.  So, following should be more correct... 

find src -name .git -type d | xargs rm -rf 

Somehow while the following sort of works, it errors out with "No such file or directory" yet it does remove the directory.  It's due to find having the results of .git contents and removing those.  So, use the pipe version above instead.

find src -name .git -type d -exec rm -rf {} \;